### PR TITLE
added request-resend-email endpoint

### PR DESCRIPTION
**Pull request by jainsvaibhav**

### CODE REVIEW

**Comment by jainsvaibhav** [comment]
please ignore the first three commits in this branch, bc the same commits are included into merge api and authentication app and endpoint pr. @waltage @adelaidemz @yalexublue

**Review by waltage — CHANGES_REQUESTED** [review]
Make changes to email link to be actual <a href='......>MESSAGE FOR LINK</a> etc. (both here and in the original email

File: src/backend/server/authentication/views.py [hunk 1]
```diff
@@ -80,24 +84,60 @@ class VerifyEmail(views.APIView):
...
+			    "Hi there"
+			    + ", "
+			    + "\n\nUse the link below to re-verify your email \n"
+			    + absurl
+			)
+			token = PasswordResetTokenGenerator().make_token(user)
+			data = {
+			    "email_body": email_body,
+			    "to_email": user.email,
+			    "email_subject": "Activate your account",
```
Comment (jainsvaibhav) [inline src/backend/server/authentication/views.py]: change the email to a href and click button

**Comment by jainsvaibhav** [comment]
Merging it to develop based on the OH discussion

**Pull request merged**